fili-core/src/main/java/com/yahoo/bard/webservice/data/DruidResponseParser.java

```diff
@@ -305,4 +308,16 @@ private Object getNodeValue(JsonNode node) {
                 node.isNull() ? null :
                 node;
     }
+
+    /**
+     * Produce the schema-defining columns for a given druid query.
+     * Pass through to druid query to allow for possible behavior customization on injected DruidResponseParsers.
```

Comment:
This is an implementation detail, right? Shouldn't this go _inside_ the method, since it doesn't really impact it's contract?